sdks/python/README.md

```diff
@@ -26,7 +26,7 @@ import yaml
 
 import argo_workflows
 from argo_workflows.api import workflow_service_api
-from argo_workflows.models.io_argoproj_workflow_v1alpha1_workflow_create_request import \
+from argo_workflows.model.io_argoproj_workflow_v1alpha1_workflow_create_request import \
```

Comment:
“model” is imported in “models” module already so I am not sure if this change is necessary.

Reply:
I lack the full context, but the import simply breaks when running the code, while with `model` it does not. I also haven't found any examples of importing with `models` in the code base.